server/src/db/migrations/20220203144345_createReviews.cjs

```diff
@@ -0,0 +1,30 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+    return knex.schema.createTable('reviews', (table) => {
+        table.bigIncrements('id')
```

Comment:
Leaving the question here - typically a review can only be created by an authenticated user, rather than just any visitor. But that's a design decision you can discuss, or add later!

Reply:
Im gonna re-up on this and actually have you update your code so that a user is related to the review that is made. I realize that this is a design decision, but its a valuable workflow that is going to be of use to you in later features and in designing your BT. Make it so that the currently logged in user's id is added to a review record